projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -18,95 +23,124 @@
+            selectedSuggestion: suggestion,
+            searchSuggestions: result.suggestions
+          });
+          updateChosenWord(suggestion)
+        "
+      >
+      </a>
+    </li>
+  </ul>
 
-  <div class="products" *ngIf="result.products">
-    <a
-      *ngFor="let product of result.products"
-      [routerLink]="
-        {
-          cxRoute: 'product',
-          params: product
-        } | cxUrl
-      "
-      [class.has-media]="config.displayProductImages"
-      (keydown.arrowup)="focusPreviousChild($event)"
-      (keydown.arrowdown)="focusNextChild($event)"
-      (keydown.enter)="close($event, true)"
-      (keydown.escape)="close($event, true)"
-      (blur)="close($event)"
-      (mousedown)="preventDefault($event)"
-      (click)="
-        dispatchProductEvent({
-          freeText: searchInput.value,
-          productCode: product.code
-        })
-      "
-    >
-      <cx-media
-        *ngIf="config.displayProductImages"
-        [container]="product.images?.PRIMARY"
-        format="thumbnail"
-        [alt]="product.summary"
-      ></cx-media>
-      <h4 class="name" [innerHTML]="product.nameHtml"></h4>
-      <span class="price">{{ product.price?.formattedValue }}</span>
-    </a>
+  <ul
+    class="products"
+    *ngIf="result.products"
+    attr.aria-label="{{ 'searchBox.ariaLabelProducts' | cxTranslate }}"
+    tabindex="0"
+  >
+    <li *ngFor="let product of result.products">
+      <a
+        [routerLink]="
+          {
+            cxRoute: 'product',
+            params: product
+          } | cxUrl
+        "
+        [class.has-media]="config.displayProductImages"
+        (keydown.arrowup)="focusPreviousChild($event)"
+        (keydown.arrowdown)="focusNextChild($event)"
+        (keydown.enter)="close($event, true)"
+        (keydown.escape)="close($event, true)"
+        (blur)="close($event)"
+        (mousedown)="preventDefault($event)"
+        (click)="
+          dispatchProductEvent({
+            freeText: searchInput.value,
+            productCode: product.code
+          })
+        "
+      >
+        <cx-media
+          *ngIf="config.displayProductImages"
+          [container]="product.images?.PRIMARY"
+          format="thumbnail"
+          alt="test"
+          role="presentation"
+        ></cx-media>
+        <div class="name" [innerHTML]="product.nameHtml"></div>
+        <span class="price">{{ product.price?.formattedValue }}</span>
+      </a>
+    </li>
+  </ul>
+  <span id="initialDescription" class="cx-visually-hidden">
+    {{ 'searchBox.initialDescription' | cxTranslate }}
+  </span>
+  <div
+    *ngIf="result.suggestions?.length || result.products?.length"
+    aria-live="assertive"
+    class="cx-visually-hidden"
+  >
+    {{ result.suggestions?.length }}
+    {{ 'searchBox.suggestionsAnd' | cxTranslate }}
+    {{ result.products?.length }}
+    {{ 'searchBox.productsAreAvailable' | cxTranslate }}
+    {{ 'searchBox.initialDescription' | cxTranslate }}
```

Comment:
```suggestion
    {{ 'searchBox.result' | cxTranslate : { suggestionsCount: result.suggestions?.length, productsCount: result.products?.length } }}
```

Reply:
fixed and left `initialDescription` as is because it's a separate sentence and purpose